### ISSUE TITLE
mamba clean crash

### ISSUE THREAD

**Issue by ThomasHezard** [mamba-org/mamba#3044]
### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### Anaconda default channels

- [X] I do NOT use the Anaconda default channels (pkgs/* etc.)

### How did you install Mamba?

Mambaforge or latest Miniforge

### Search tried in issue tracker

clean, crash

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

I do not have this problem with Conda, just with Mamba

### Describe your issue

The command `mamba clean` ends with error for all the options I tried: 
- `mamba clean -a`
- `mamba clean -i`
- `mamba clean -p`
- `mamba clean -t`
- `mamba clean -l`

The error is always the same: 
`No module named 'conda.cliconda'` 
followed by an error report (see below)

Problem is the same with or without an activated environement.

`conda clean` works fine with all options.

My `miniforge3` install is only a few days old, no plugins, I just created one simple env and and run `mamba update conda mamba` once.

### mamba info / micromamba info

```shell
mamba version : 1.5.4
     active environment : None
            shell level : 0
       user config file : /Users/thomashezard/.condarc
 populated config files : /Users/thomashezard/opt/miniforge3/.condarc
          conda version : 23.11.0
    conda-build version : not installed
         python version : 3.10.13.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=m1
                          __conda=23.11.0=0
                          __osx=14.1.1=0
                          __unix=0=0
       base environment : /Users/thomashezard/opt/miniforge3  (writable)
      conda av data dir : /Users/thomashezard/opt/miniforge3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/osx-arm64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /Users/thomashezard/opt/miniforge3/pkgs
                          /Users/thomashezard/.conda/pkgs
       envs directories : /Users/thomashezard/opt/miniforge3/envs
                          /Users/thomashezard/.conda/envs
               platform : osx-arm64
             user-agent : conda/23.11.0 requests/2.31.0 CPython/3.10.13 Darwin/23.1.0 OSX/14.1.1 solver/libmamba conda-libmamba-solver/23.11.1 libmambapy/1.5.4
                UID:GID : 501:20
             netrc file : None
           offline mode : False
```

### Logs

NB: I purposefully removed some sensitive env variables, but I'm almost 100% sure these have nothing to do with the issue here.

```shell
# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "/Users/thomashezard/opt/miniforge3/lib/python3.10/site-packages/conda/exception_handler.py", line 17, in __call__
        return func(*args, **kwargs)
      File "/Users/thomashezard/opt/miniforge3/lib/python3.10/site-packages/mamba/mamba.py", line 956, in exception_converter
        raise e
      File "/Users/thomashezard/opt/miniforge3/lib/python3.10/site-packages/mamba/mamba.py", line 949, in exception_converter
        exit_code = _wrapped_main(*args, **kwargs)
      File "/Users/thomashezard/opt/miniforge3/lib/python3.10/site-packages/mamba/mamba.py", line 895, in _wrapped_main
        result = do_call(parsed_args, p)
      File "/Users/thomashezard/opt/miniforge3/lib/python3.10/site-packages/mamba/mamba.py", line 758, in do_call
        exit_code = clean(args, parser)
      File "/Users/thomashezard/opt/miniforge3/lib/python3.10/site-packages/mamba/mamba.py", line 701, in clean
        module = import_module("conda.cli" + relative_mod, __name__.rsplit(".", 1)[0])
      File "/Users/thomashezard/opt/miniforge3/lib/python3.10/importlib/__init__.py", line 126, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
      File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
      File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
      File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
      File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
      File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
      File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
    ModuleNotFoundError: No module named 'conda.cliconda'

`$ /Users/thomashezard/opt/miniforge3/bin/mamba clean -a`

  environment variables:
                 CIO_TEST=<not set>
        CONDA_DEFAULT_ENV=base
                CONDA_EXE=/Users/thomashezard/opt/miniforge3/bin/conda
             CONDA_PREFIX=/Users/thomashezard/opt/miniforge3
    CONDA_PROMPT_MODIFIER=(base)
         CONDA_PYTHON_EXE=/Users/thomashezard/opt/miniforge3/bin/python
               CONDA_ROOT=/Users/thomashezard/opt/miniforge3
              CONDA_SHLVL=1
           CURL_CA_BUNDLE=<not set>
        DYLD_LIBRARY_PATH=/Users/thomashezard/usr/lib:/opt/homebrew/lib:
                 INFOPATH=/opt/homebrew/share/info:
               LD_PRELOAD=<not set>
                  MANPATH=/Users/thomashezard/bash_scripts/git-
                          subrepo/man:/opt/homebrew/share/man::
                     PATH=/Users/thomashezard/opt/miniforge3/bin:/Users/thomashezard/opt/google-
                          cloud-sdk/bin:/Users/thomashezard/opt/miniforge3/condabin:/Users/thoma
                          shezard/Library/Android/sdk/cmdline-tools/9.0/bin:/Users/thomashezard/
                          Library/Android/sdk/emulator:/Users/thomashezard/Library/Android/sdk/t
                          ools:/Users/thomashezard/Library/Android/sdk/tools/bin:/Users/thomashe
                          zard/Library/Android/sdk/platform-tools:/Users/thomashezard/.cargo/bin
                          :/Users/thomashezard/bash_scripts/git-subrepo/lib:/Users/thomashezard/
                          .pyenv/shims:/Users/thomashezard/.pyenv/bin:/Users/thomashezard/.jenv/
                          shims:/Users/thomashezard/.rbenv/shims:/opt/homebrew/bin:/opt/homebrew
                          /sbin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/
                          sbin:/sbin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap
                          /usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/boots
                          trap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstr
                          ap/usr/appleinternal/bin:/Library/Apple/usr/bin:/Applications/VMware F
                          usion.app/Contents/Public:/usr/local/share/dotnet:~/.dotnet/tools:/Use
                          rs/thomashezard/.maestro/bin:/Users/thomashezard/usr/bin:/usr/local/bi
                          n:/Users/thomashezard/bash_scripts:/Users/thomashezard/opt/depot_tools
                          :/Applications/Visual Studio Code.app/Contents/Resources/app/bin:/User
                          s/thomashezard/Library/Application Support/JetBrains/Toolbox/scripts
       REQUESTS_CA_BUNDLE=<not set>
            SSL_CERT_FILE=<not set>

     active environment : base
    active env location : /Users/thomashezard/opt/miniforge3
            shell level : 1
       user config file : /Users/thomashezard/.condarc
 populated config files : /Users/thomashezard/opt/miniforge3/.condarc
          conda version : 23.11.0
    conda-build version : not installed
         python version : 3.10.13.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=m1
                          __conda=23.11.0=0
                          __osx=14.1.1=0
                          __unix=0=0
       base environment : /Users/thomashezard/opt/miniforge3  (writable)
      conda av data dir : /Users/thomashezard/opt/miniforge3/etc/conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/osx-arm64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : /Users/thomashezard/opt/miniforge3/pkgs
                          /Users/thomashezard/.conda/pkgs
       envs directories : /Users/thomashezard/opt/miniforge3/envs
                          /Users/thomashezard/.conda/envs
               platform : osx-arm64
             user-agent : conda/23.11.0 requests/2.31.0 CPython/3.10.13 Darwin/23.1.0 OSX/14.1.1 solver/libmamba conda-libmamba-solver/23.11.1 libmambapy/1.5.4
                UID:GID : 501:20
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.
If you suspect this error is being caused by a malfunctioning plugin,
consider using the --no-plugins option to turn off plugins.

Example: conda --no-plugins install <package>

Alternatively, you can set the CONDA_NO_PLUGINS environment variable on
the command line to run the command without plugins enabled.

Example: CONDA_NO_PLUGINS=true conda install <package>
```

### ~/.condarc

```yaml
no .condarc for now
```

**Comment by rohit-guitar** [comment]
This is still happening to me inside Docker while building the container. Mamba version 1.5.4

`
#33 193.3 No module named 'conda.cliconda'
#33 193.7
#33 193.7 # >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<
#33 193.7
#33 193.7     Traceback (most recent call last):
#33 193.7       File "/opt/conda/lib/python3.8/site-packages/conda/exception_handler.py", line 17, in __call__
#33 193.7         return func(*args, **kwargs)
#33 193.7       File "/opt/conda/lib/python3.8/site-packages/mamba/mamba.py", line 956, in exception_converter
#33 193.7         raise e
#33 193.7       File "/opt/conda/lib/python3.8/site-packages/mamba/mamba.py", line 949, in exception_converter
#33 193.7         exit_code = _wrapped_main(*args, **kwargs)
#33 193.7       File "/opt/conda/lib/python3.8/site-packages/mamba/mamba.py", line 895, in _wrapped_main
#33 193.7         result = do_call(parsed_args, p)
#33 193.7       File "/opt/conda/lib/python3.8/site-packages/mamba/mamba.py", line 758, in do_call
#33 193.7         exit_code = clean(args, parser)
#33 193.7       File "/opt/conda/lib/python3.8/site-packages/mamba/mamba.py", line 701, in clean
#33 193.7         module = import_module("conda.cli" + relative_mod, __name__.rsplit(".", 1)[0])
#33 193.7       File "/opt/conda/lib/python3.8/importlib/__init__.py", line 127, in import_module
#33 193.7         return _bootstrap._gcd_import(name[level:], package, level)
#33 193.7       File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
#33 193.7       File "<frozen importlib._bootstrap>", line 991, in _find_and_load
#33 193.7       File "<frozen importlib._bootstrap>", line 961, in _find_and_load_unlocked
#33 193.7       File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
#33 193.7       File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
#33 193.7       File "<frozen importlib._bootstrap>", line 991, in _find_and_load
#33 193.7       File "<frozen importlib._bootstrap>", line 961, in _find_and_load_unlocked
#33 193.7       File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
#33 193.7       File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
#33 193.7       File "<frozen importlib._bootstrap>", line 991, in _find_and_load
#33 193.7       File "<frozen importlib._bootstrap>", line 973, in _find_and_load_unlocked
#33 193.7     ModuleNotFoundError: No module named 'conda.cliconda'
#33 193.7
#33 193.7 `$ /opt/conda/bin/mamba clean -yaf`
#33 193.7
#33 193.7   environment variables:
#33 193.7                  CIO_TEST=<not set>
#33 193.7                CONDA_ROOT=/opt/conda
#33 193.7            CURL_CA_BUNDLE=<not set>
#33 193.7           LD_LIBRARY_PATH=/usr/local/nvidia/lib:/usr/local/nvidia/lib64
#33 193.7                LD_PRELOAD=<not set>
#33 193.7                      PATH=/opt/conda/bin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sb
#33 193.7                           in:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
#33 193.7              PYTHON_MAJOR=3
#33 193.7              PYTHON_MINOR=8
#33 193.7              PYTHON_PATCH=17
#33 193.7        REQUESTS_CA_BUNDLE=<not set>
#33 193.7             SSL_CERT_FILE=<not set>
#33 193.7
#33 193.7      active environment : None
#33 193.7        user config file : /home/test/.condarc
#33 193.7  populated config files : /opt/conda/.condarc
#33 193.7           conda version : 23.11.0
#33 193.7     conda-build version : not installed
#33 193.7          python version : 3.8.17.final.0
#33 193.7                  solver : libmamba (default)
#33 193.7        virtual packages : __archspec=1=x86_64_v4
#33 193.7                           __conda=23.11.0=0
#33 193.7                           __glibc=2.17=0
#33 193.7                           __linux=6.1.32=0
#33 193.7                           __unix=0=0
#33 193.7        base environment : /opt/conda  (writable)
#33 193.7       conda av data dir : /opt/conda/etc/conda
#33 193.7   conda av metadata url : None
#33 193.7            channel URLs : https://conda.anaconda.org/conda-forge/linux-64
#33 193.7                           https://conda.anaconda.org/conda-forge/noarch
#33 193.7           package cache : /opt/conda/pkgs
#33 193.7                           /home/test/.conda/pkgs
#33 193.7        envs directories : /opt/conda/envs
#33 193.7                           /home/test/.conda/envs
#33 193.7                platform : linux-64
#33 193.7              user-agent : conda/23.11.0 requests/2.31.0 CPython/3.8.17 Linux/6.1.32-0-virt oracle/7.9 glibc/2.17 solver/libmamba conda-libmamba-solver/23.5.0 libmambapy/1.4.1
#33 193.7                 UID:GID : 1000:100
#33 193.7              netrc file : None
#33 193.7            offline mode : False
#33 193.7
#33 193.7
#33 193.7 An unexpected error has occurred. Conda has prepared the above report.
#33 193.7 If you suspect this error is being caused by a malfunctioning plugin,
#33 193.7 consider using the --no-plugins option to turn off plugins.
#33 193.7
#33 193.7 Example: conda --no-plugins install <package>
#33 193.7
#33 193.7 Alternatively, you can set the CONDA_NO_PLUGINS environment variable on
#33 193.7 the command line to run the command without plugins enabled.
#33 193.7
#33 193.7 Example: CONDA_NO_PLUGINS=true conda install <package>
#33 193.7
#33 ERROR: process "/bin/sh -c mamba env update -n root -f /opt/jupyter.yaml && mamba clean -yaf && rm -rf /home/test/.cache/pip" did not complete successfully: exit code: 1
------
 > [29/80] RUN mamba env update -n root -f /opt/jupyter.yaml && mamba clean -yaf && rm -rf /home/test/.cache/pip:
193.7 If you suspect this error is being caused by a malfunctioning plugin,
193.7 consider using the --no-plugins option to turn off plugins.
193.7
193.7 Example: conda --no-plugins install <package>
193.7
193.7 Alternatively, you can set the CONDA_NO_PLUGINS environment variable on
193.7 the command line to run the command without plugins enabled.
193.7
193.7 Example: CONDA_NO_PLUGINS=true conda install <package>
193.7
------
Dockerfile.gpu:192

`

**Comment by rohit-guitar** [comment]
This is still happening to me inside Docker while building the container. Mamba version 1.5.4, conda 23.11.0

**Comment by ThomasHezard** [comment]
@rohit-guitar    
Fix #3040 cited above has been merged but not released yet. Should be included in next release I guess.

**Comment by ReimarBauer** [comment]
I have the same problem in our docker container for linux and mac but.
It happens also on windows, but there somehow it can continue.

```
Looking for: ['mss', 'python']


Downloading and Extracting Packages: ...working... done
Preparing transaction: ...working... done
Verifying transaction: ...working... done
Executing transaction: ...working... done
No module named 'conda.cliconda'

# >>>>>>>>>>>>>>>>>>>>>> ERROR REPORT <<<<<<<<<<<<<<<<<<<<<<

    Traceback (most recent call last):
      File "C:\Users\runneradmin\miniforge\lib\site-packages\conda\exception_handler.py", line 17, in __call__
        return func(*args, **kwargs)
      File "C:\Users\runneradmin\miniforge\lib\site-packages\mamba\mamba.py", line 956, in exception_converter
        raise e
      File "C:\Users\runneradmin\miniforge\lib\site-packages\mamba\mamba.py", line 949, in exception_converter
        exit_code = _wrapped_main(*args, **kwargs)
      File "C:\Users\runneradmin\miniforge\lib\site-packages\mamba\mamba.py", line 895, in _wrapped_main
        result = do_call(parsed_args, p)
      File "C:\Users\runneradmin\miniforge\lib\site-packages\mamba\mamba.py", line 758, in do_call
        exit_code = clean(args, parser)
      File "C:\Users\runneradmin\miniforge\lib\site-packages\mamba\mamba.py", line 701, in clean
        module = import_module("conda.cli" + relative_mod, __name__.rsplit(".", 1)[0])
      File "C:\Users\runneradmin\miniforge\lib\importlib\__init__.py", line 126, in import_module
        return _bootstrap._gcd_import(name[level:], package, level)
      File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
      File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
      File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
      File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
      File "<frozen importlib._bootstrap>", line 992, in _find_and_load_unlocked
      File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
      File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
      File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
      File "<frozen importlib._bootstrap>", line 1004, in _find_and_load_unlocked
                          Server\140\DTS\Binn;C:\Program Files\Microsoft SQL
                          Server\150\DTS\Binn;C:\Program Files\Microsoft SQL Server\160\DTS\Binn
                          ;C:\Strawberry\c\bin;C:\Strawberry\perl\site\bin;C:\Strawberry\perl\bi
                          n;C:\ProgramData\chocolatey\lib\pulumi\tools\Pulumi\bin;C:\Program
                          Files\TortoiseSVN\bin;C:\Program
                          Files\CMake\bin;C:\ProgramData\chocolatey\lib\maven\apache-
                          maven-3.8.7\bin;C:\Program Files\Microsoft Service
                          Fabric\bin\Fabric\Fabric.Code;C:\Program Files\Microsoft SDKs\Service
                          Fabric\Tools\ServiceFabricLocalClusterManager;C:\Program
                          Files\nodejs;C:\Program Files\Git\cmd;C:\Program
                          Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Program
                          Files\GitHub CLI;c:\tools\php;C:\Program Files
                          (x86)\sbt\bin;C:\Program Files\Amazon\AWSCLIV2;C:\Program
                          Files\Amazon\SessionManagerPlugin\bin;C:\Program
                          Files\Amazon\AWSSAMCLI\bin;C:\Program Files\Microsoft SQL
                          Server\130\Tools\Binn;C:\Program Files\LLVM\bin;C:\Users\runneradmin\.
                          dotnet\tools;C:\Users\runneradmin\.cargo\bin;C:\Users\runneradmin\AppD
                          ata\Local\Microsoft\WindowsApps;C:\Users\runneradmin\Miniforge\condabi
                          n
PSMODULEANALYSISCACHEPATH=C:\PSModuleAnalysisCachePath\ModuleAnalysisCache
             PSMODULEPATH=C:\\Modules\azurerm_2.1.0;C:\\Modules\azure_2.1.0;C:\Users\packer\Docu
                          ments\WindowsPowerShell\Modules;C:\Program Files\WindowsPowerShell\Mod
                          ules;C:\Windows\system32\WindowsPowerShell\v1.0\Modules;C:\Program
                          Files\Microsoft SQL Server\130\Tools\PowerShell\Modules\
       REQUESTS_CA_BUNDLE=<not set>
        SELENIUM_JAR_PATH=C:\selenium\selenium-server.jar
            SSL_CERT_FILE=<not set>

     active environment : mssenv
    active env location : C:\Users\runneradmin\miniforge\envs\mssenv
            shell level : 1
       user config file : C:\Users\runneradmin\.condarc
 populated config files : C:\Users\runneradmin\miniforge\.condarc
          conda version : 23.11.0
    conda-build version : not installed
         python version : 3.10.12.final.0
                 solver : libmamba (default)
       virtual packages : __archspec=1=x86_64
                          __conda=23.11.0=0
                          __win=0=0
       base environment : C:\Users\runneradmin\miniforge  (writable)
      conda av data dir : C:\Users\runneradmin\miniforge\etc\conda
  conda av metadata url : None
           channel URLs : https://conda.anaconda.org/conda-forge/win-64
                          https://conda.anaconda.org/conda-forge/noarch
          package cache : C:\Users\runneradmin\miniforge\pkgs
                          C:\Users\runneradmin\.conda\pkgs
                          C:\Users\runneradmin\AppData\Local\conda\conda\pkgs
       envs directories : C:\Users\runneradmin\miniforge\envs
                          C:\Users\runneradmin\.conda\envs
                          C:\Users\runneradmin\AppData\Local\conda\conda\envs
               platform : win-64
             user-agent : conda/23.11.0 requests/2.31.0 CPython/3.10.12 Windows/10 Windows/10.0.20348 solver/libmamba conda-libmamba-solver/23.11.1 libmambapy/1.5.4
          administrator : True
             netrc file : None
           offline mode : False


An unexpected error has occurred. Conda has prepared the above report.
If you suspect this error is being caused by a malfunctioning plugin,
consider using the --no-plugins option to turn off plugins.

Example: conda --no-plugins install <package>

Alternatively, you can set the CONDA_NO_PLUGINS environment variable on
the command line to run the command without plugins enabled.

Example: CONDA_NO_PLUGINS=true conda install <package>

mss                       8.3.2           py310h5588dad_0    conda-forge
Done! To start msui from the MSS Software: Press the Windows button, type in "msui" and press enter.
Alternatively:
1. Activate your mamba environment with this command: "mamba activate mssenv"
2. Start msui with this command: "msui"
Press any key to continue . . . 
```

**Comment by ReimarBauer** [comment]
### Workaround 

instead of  `mamba clean --all -y ` we can use `conda clean --all -y`

**Comment by Shamik-07** [comment]
yes the 

> ### Workaround
> instead of `mamba clean --all -y ` we can use `conda clean --all -y`

yes this certainly works but the error  is an upstream error for Linux systems.

**Comment by ReimarBauer** [comment]
It fails also on Windows. Somehow it continues there which I assume is not correct. Crashes or Exceptions should behave similiar.

**Comment by JohanMabille** [comment]
I've released mamba 1.5.5 with the fix included.

**Comment by Shamik-07** [comment]
> I've released mamba 1.5.5 with the fix included.

Will check it once we can update it via `mamba update -n base mamba`

**Comment by ReimarBauer** [comment]
> I've released mamba 1.5.5 with the fix included.

This works for us. Thx.

**Comment by Shamik-07** [comment]
@ReimarBauer maybe you can close this issue, if it's fixed for you.

**Comment by ThomasHezard** [comment]
Fixed with the 1.5.5 release, many thanks 🙏

**Comment by ThomasHezard** [comment]
Fixed with the 1.5.5 release, many thanks 🙏
